src/main.rs

```diff
@@ -1,5 +1,6 @@
 mod command;
 mod mv;
+mod post_build_process;
```

Comment:
module名をコマンド名にあわせてbuildとかにしてほしいです！
```suggestion
mod build;
```